examples/doc_fitting_emcee.py

```diff
@@ -53,6 +53,17 @@ def residual(p):
                                  truths=list(res.params.valuesdict().values()))
     plt.show()
 
+if HASPYLAB:
+    plt.plot(res.acceptance_fraction)
+    plt.xlabel('walker')
+    plt.ylabel('acceptance fraction')
+    plt.show()
+
```

Comment:
something along these lines could/should probably be added to ```fitting.rst``` as well so that it shows up in the documentation.

Reply:
I have done that in the last commit